src/NavList/NavList.tsx

```diff
@@ -235,18 +236,18 @@ Divider.displayName = 'NavList.Divider'
 // ----------------------------------------------------------------------------
 // NavList.Group
 
-export type NavListGroupProps = {
+export type NavListGroupProps = Omit<ActionListGroupProps, 'selectionVariant' | 'variant' | 'auxiliaryText'> & {
```

Comment:
Could you update the props documentation for NavList.Group to show that props are passed through to ActionList.Group? This should do the trick:

```
...
### NavList.Group

<PropsTable>
  ...
  <PropsTablePassthroughPropsRow
    elementName="ActionList.Group"
    passthroughPropsLink={
      <>
        the <Link href="/ActionList#actionlistgroup">ActionList.Group docs</Link>
      </>
    }
  />
</PropsTable>
```

It should render something similar to this: 
<img width="819" alt="CleanShot 2022-06-20 at 11 13 56@2x" src="https://user-images.githubusercontent.com/4608155/174658484-8a5e5ad7-e203-4d19-a17b-8d57a8832b16.png">

Reply:
can do